### ISSUE TITLE
Allow `filter` to accept a single suite or spec

### ISSUE THREAD

**Issue by parkji** [parkji/grunt-jasmine-webpack#7]

**Comment by davidnormo** [comment]
This would be excellent! At the moment I see the following when filtering:

```
Running "jasmine_webpack:dev" (jasmine_webpack) task
Results: 0/0 passed.

Done, without errors.
```

**Comment by parkji** [comment]
@davidnormo This has been fixed in the latest release, 0.4.0.

**Comment by davidnormo** [comment]
Great thanks!
